src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +228,28 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('exit-control');
+    if (!Object.values(BUTTON_TYPES).includes(option)) {
+      return;
+    }
+
+    const button = this.doc_.createElement('a');
```

Comment:
If we're creating a button, let's use the `button` element :)) It has a lof of accessibility features coming for free, and is semantically correct.

Reply:
can you add the `role="button"` here like we mentioned?

Reply:
I converted it to a button so it doesn't need to be migrated to a button tag later on.